utils/driver_options/driveroptions.cpp

```diff
@@ -63,16 +63,16 @@ DriverOptions::DriverOptions() {
         return receive_channels;
     }();
 
-/*    radar_control_to_driver_address_ = config_pt.get<std::string>
-                                            ("radar_control_to_driver_address");
-    driver_to_rx_dsp_address_ = config_pt.get<std::string>("driver_to_rx_dsp_address");*/
     router_address_ = config_pt.get<std::string>("router_address");
     driver_to_radctrl_identity_ = config_pt.get<std::string>("driver_to_radctrl_identity");
     driver_to_dsp_identity_ = config_pt.get<std::string>("driver_to_dsp_identity");
     driver_to_brian_identity_ = config_pt.get<std::string>("driver_to_brian_identity");
     radctrl_to_driver_identity_ = config_pt.get<std::string>("radctrl_to_driver_identity");
     dsp_to_driver_identity_ = config_pt.get<std::string>("dsp_to_driver_identity");
     brian_to_driver_identity_ = config_pt.get<std::string>("brian_to_driver_identity");
+    ringbuffer_name_ = config_pt.get<std::string>("ringbuffer_name");
+    ringbuffer_size_bytes_ = boost::lexical_cast<double>(
```

Comment:
why does this need to be a double?

Reply:
So I can use scientific notation in the config file.